resources/lang/en.json

```
@@ -595,5 +595,83 @@
     "redirecting": "You are being redirected...",
     "not_authorized": "You are not authorized to access this website.",
     "contact_admin": "If you believe you are seeing this message in error, please contact the website administrator."
+  },
+  "flag_input": {
+    "title": "Flag Selector",
+    "real": "Real Flags",
+    "custom": "Custom Flags",
+    "select_layer": "Select a Layer",
+    "max_layer_reached_1": "You've reached the maximum number of layers.",
+    "max_layer_reached_2": "Please remove some to add new ones.",
+    "preview": "Preview",
+    "apply": "Apply Custom Flag",
+    "layer": "Layer",
+    "fixed": "fixed",
+    "remove": "Remove",
+    "color": "Color",
+    "paste_ctmfg_placeholder": "Paste ctmfg code here...",
```

What does ctmfg mean?
```suggestion
    "paste_ctmfg_placeholder": "Paste custom code here...",
```